docker-compose.yml

```diff
@@ -72,7 +71,7 @@ services:
   broker-rabbitmq:
     image: rabbitmq:3-management
     ports:
-      - 61613:61613
```

Comment:
Port `61613` is used inside the code. The pipeline probably broke because of this. Do you really need to change the port?
The idea would be to start one container at a time and not both at the same time.
Did this configuration work locally?

Reply:
I started all the services at the same time (using `docker compose up [--build]`), hence the reason it was reporting a port already in use error. I can:

1. revert the change and leave clear guidelines on the README about how to run the tests in docker compose setup, OR
2. change the rabbitmq port in the code to support both cases.

I am not sure how detrimental the 2nd option will be, but a person new to the repo like I am will expect it not to fail if they ran the same command.

Reply:
There's this [section](https://github.com/juntossomosmais/django-stomp/tree/b8a1a5807eb3c3c0d7be303159646fe7ddfd336f?tab=readme-ov-file#tests) that says to run the tests with either  `broker-activemq` or `broker-rabbitmq`. If it's not clear enough, it's an opportunity to improve it

Reply:
> There's this [section](https://github.com/juntossomosmais/django-stomp/tree/b8a1a5807eb3c3c0d7be303159646fe7ddfd336f?tab=readme-ov-file#tests) that says to run the tests with either `broker-activemq` or `broker-rabbitmq`. If it's not clear enough, it's an opportunity to improve it

I'll have a closer look at how to better approach this, then update ASAP.

Reply:
@ricardochaves , I am not sure why the pipeline is still failing despite rolling back the port change. I successfully ran `tox` in my local setup as per the documentation. Checking the Azure pipeline reports seems to require privileged access, or I may not be going about it correctly.